src/fu-util-common.h

```diff
@@ -14,6 +14,11 @@
 
 #include "fu-console.h"
 
+/* custom return codes */
+#define EXIT_NOTHING_TO_DO 2
+#define EXIT_NOT_FOUND	   3
+#define EXIT_NOT_REACHABLE ENETUNREACH
```

Comment:
Where's that defined? Could we define it locally perhaps?

Reply:
Linux kernel headers.  `errno.h`  Do you think we still need to declare locally?

Reply:
I decided to make a local copy; that's at least cleaner when it comes to building windows and mac.

Reply:
Yup, the latter two were the ones I was worried about.